website/addons/wiki/utils.py

```diff
@@ -154,3 +156,78 @@ def format_wiki_version(version, num_versions, allow_preview):
         raise InvalidVersionError
 
     return version
+
+def get_wiki_permission(node):
```

Comment:
I propose this method be elimated. Instead, use `node.get_addon('wiki').is_publicly_editable`.

The passing around of "magic values" (`"public"` and `"private"`) should be as limited as possible anyhow.